osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/application/config/AbstractOsgpSchedulerConfig.java

```diff
@@ -48,4 +70,51 @@ public OsgpScheduler osgpScheduler() throws SchedulerException {
         final Scheduler quartzScheduler = this.constructAndStartQuartzScheduler(schedulingConfigProperties);
         return new OsgpScheduler(quartzScheduler);
     }
+
+    private void testIfQuartzTablesExist() {
+        final SingleConnectionDataSource singleConnectionDataSource = new SingleConnectionDataSource();
+        singleConnectionDataSource.setDriverClassName(this.databaseDriver);
+        singleConnectionDataSource.setUsername(this.databaseUsername);
+        singleConnectionDataSource.setPassword(this.databasePassword);
+        singleConnectionDataSource.setUrl(this.getDatabaseUrl());
+        singleConnectionDataSource.setSuppressClose(false);
+
+        try {
+            final Connection connection = singleConnectionDataSource.getConnection();
+            final Statement statement = connection.createStatement();
+            statement.executeQuery("SELECT * FROM qrtz_locks LIMIT 1");
+            connection.close();
+        } catch (final SQLException e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1166)